src/interpreter/executor/interpret.js

```diff
@@ -0,0 +1,332 @@
+"use strict";
```

Comment:
This is already implicitly added when we build through Babel, no need to include this explicitly.